apps/perf-test-react-components/webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const resources = require('../../scripts/webpack/webpack-resources');
+
+// The issue here is making readable Flamegraphs that don't have complicated paths like:
+//  ~Fabric.../../packages/react/lib/components/DetailsList/DetailsRow.base.js.DetailsRowBase.render
+// But rather show paths like:
+//  ~DetailsRowBase.render (22.16%, 168 samples)
+// The only way found to do this so far has been to use a webpack serve config for bundling.
+// TODO: Should root cause why this only works as a serve config.
+module.exports = resources.createServeConfig(
```

Comment:
**suggestion:** not 100% sure if it will work. but maybe we can leverage new DX here ? 

meaning: to use ts path aliases to build so webpack builds what is needed ? thus the need to run `lage` could be completely removed from the pipeline.

NOTE: We can address this in separate PR

Reply:
I'll give it a shot in a separate PR!